proto/cron/tx.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+package neutron.cron;
+
+// this line is used by starport scaffolding # proto/tx/import
+
+option go_package = "github.com/neutron-org/neutron/x/cron/types";
+
+// Msg defines the Msg service.
+service Msg {
+// this line is used by starport scaffolding # proto/tx/rpc
```

Comment:
You sure everything's right here?

Reply:
Well we do not allow for direct messages for Cron module, that's why it's empty